docs/content/latest/quick-start/build-apps/java/ysql-ebeans.md

```diff
@@ -0,0 +1,390 @@
+
+```sql
+ysql_ebeans=# select * from users;
+```
+
+```output
+ user_id | first_name | last_name |      user_email
+---------+------------+-----------+----------------------
+       1 | John       | Smith     | jsmith@example.com
+     101 | Tom        | Stewart   | tstewart@example.com
+(2 rows)
+```
+
+```sql
+ysql_ebeans=# select * from products;
+```
+
+```output
+ product_id |    description    | price | product_name
+------------+-------------------+-------+--------------
+          1 | 200 page notebook |  7.50 | Notebook
+          2 | Mechanical pencil |  2.50 | Pencil
+(2 rows)
+```
+
+Create 2 orders using the `userId` for John.
+
+```sh
+$ curl \
+  --data '{ "userId": "1", "products": [ { "productId": 1, "units": 2 } ] }' \
+  -v -X POST -H 'Content-Type:application/json' http://localhost:8080/orders
+```
+
+```sh
+$ curl \
+  --data '{ "userId": "1", "products": [ { "productId": 1, "units": 2 }, { "productId": 2, "units": 4 } ] }' \
+  -v -X POST -H 'Content-Type:application/json' http://localhost:8080/orders
+```
+
+## Query results
+
+### Using the YSQL shell
+
+In your YSQL shell, list the tables created by the application.
+
+```sql
+ysql_ebeans=#  \d
+```
+
+```output
+List of relations
+ Schema |          Name           |   Type   |  Owner
+--------+-------------------------+----------+----------
+ public | orderline               | table    | yugabyte
+ public | orders                  | table    | yugabyte
+ public | orders_user_id_seq      | sequence | yugabyte
+ public | products                | table    | yugabyte
+ public | products_product_id_seq | sequence | yugabyte
+ public | users                   | table    | yugabyte
+ public | users_user_id_seq       | sequence | yugabyte
+(7 rows)
+```
+
+Note the 4 tables and 3 sequences in the list above.
+
+```sql
+ysql_ebeans=# SELECT count(*) FROM users;
+```
+
+```output
+ count
+-------
+     2
+(1 row)
+```
+
+```sql
+ysql_ebeans=# SELECT count(*) FROM products;
+```
+
+```output
+ count
+-------
+     2
+(1 row)
+```
+
+```sql
+ysql_ebeans=# SELECT count(*) FROM orders;
+```
+
+```output
+ count
+-------
+     2
+(1 row)
+```
+
+```sql
+ysql_ebeans=# SELECT * FROM orderline;
+```
+
+```output
+ order_id                             | product_id | units
+--------------------------------------+------------+-------
+ 45659918-bbfd-4a75-a202-6feff13e186b |          1 |     2
+ f19b64ec-359a-47c2-9014-3c324510c52c |          1 |     2
+ f19b64ec-359a-47c2-9014-3c324510c52c |          2 |     4
+(3 rows)
+```
+
+`orderline` is a child table of the parent `orders` table, and is connected using a foreign key constraint. The `users` table are connected with `orders` using a foreign key constraint so that no order can be placed with an invalid user, and that user has to be present in the users table.
+
+### Using the REST API
+
+Verify the users, products, and orders created in the `ysql_ebeans` database from the REST API server.
```

Comment:
To use the REST API server to verify that the users, products, and orders were created in the `ysql_ebeans` database, enter the following commands.